app/presenters/api/metrics_index_presenter.rb

```diff
@@ -37,7 +37,8 @@ def create_path
   protected
 
   def raw_collection
-    @raw_collection ||= metrics? ? top_level_metrics : method_metrics
+    collection = metrics? ? top_level_metrics : method_metrics
+    @raw_collection ||= collection.includes(:proxy_rules)
```

Comment:
There is a nitpick here, that if `@raw_collection` is set, no need for the previous statement but it doesn't matter

Reply:
I assumed that's a lazy statement? No data is loaded. Feel free to suggest an improvement.